src/ape/contracts/base.py

```diff
@@ -440,6 +441,9 @@ def poll_logs(
             required_confirmations (Optional[int]): The amount of confirmations to wait
               before yielding the block. The more confirmations, the less likely a reorg will occur.
               Defaults to the network's configured required confirmations.
+            new_block_timeout (Optional[int]): The amount of time to wait for a new block before
+              quitting. Defaults to 10 seconds for local networks or ``3 * block_time`` for live
```

Comment:
Remove "Defaults" because it doesn't default here

Reply:
it is the default behavior though

Reply:
`block_time * 3` sounds insufficient, i can demonstrate:

```python
>>> block_times = chain.blocks.query('timestamp', start_block=chain.blocks.height - 10000, stop_block=chain.blocks.height).diff()
>>> block_times.describe()
          timestamp
count  10000.000000
mean      13.285700
std       12.749123
min        1.000000
25%        4.000000
50%        9.000000
75%       18.000000
max      121.000000
>>> desc = block_times.describe()['timestamp']
>>> desc['max'] / desc['mean']
9.107536674770618
```
as you can see, a worst case is 9.1x the average block time and 13.4x of the median block time